src/index.html

```diff
@@ -9,6 +9,28 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="header__logo--a">
+        <img href="#" class="header__logo"
+         src="images/logo.png"
+         alt="Logo">
+      </a>
+      <nav class="header__nav">
+        <ul class="nav__list">
+          <li><a href="#" class="nav__link is-active">Apple</a></li>
```

Comment:
Please refer to this [guideline](https://mate-academy.github.io/style-guides/htmlcss.html#html-line-wrapping) and fix the formatting